projects/demo/src/modules/components/calendar-range/examples/2/index.ts

```diff
@@ -20,5 +21,8 @@ export const calendarStream$ = of(
             useValue: calendarStream$,
         },
     ],
+    host: {
+        tuiExample: '2',
+    },
```

Comment:
it's need to prevent components collision error
https://angular.io/errors/NG0912

Reply:
removed it, because don't need it after https://github.com/taiga-family/taiga-ui/pull/7523